### DISCUSSION TITLE
export profile from uvtools to import into LycheeSlicer

### DISCUSSION THREAD

**Discussion by webtv123** [sn4k3/UVtools#816] [General]
i loaded a model into uvtool im looking for a export profile from uvtools to import into LycheeSlicer or can import A model thats been sliced in another program directly into LycheeSlicer to save the profile settings. What I'm trying to do is when you get a printer it come with test prints and these Are a general basic better settings. I would like to transfer them settings from uv tools to LycheeSlicer . as a example i would like to save
Layer Thickness (um):
Exposure Time (sec):
Bottom Curing Time 
ect ect

**Comment by sn4k3** [comment]
From what I know that is already possible.
If you referring to this:

![image](https://github.com/sn4k3/UVtools/assets/113281/aa885fbd-1b16-4770-98ab-3da67688332e)

See the next message:

![image](https://github.com/sn4k3/UVtools/assets/113281/99bf39a0-5b02-4769-9816-c0ce5e3306ff)


They allow import settings from a sliced file, eg: ctb.

**Comment by webtv123** [comment]
https://www.anycubic.com/pages/firmware-software
im needing these settings for TEST.pm3 
Layer Thickness (um):
Exposure Time (sec):
Bottom Curing Time.
i can load the TEST.pm3 in uv tools but i have to copy and paste each settings to import to LycheeSlicer.
im not talking about the resin setting. through save file?
![image](https://github.com/sn4k3/UVtools/assets/61340950/3e9919ed-36ce-4b0f-a931-bdc3d2f85e14)

how do i get this into Lycheeslicer
![image](https://github.com/sn4k3/UVtools/assets/61340950/f6874ecf-565d-4591-8475-5064319d79b6)

**Reply by sn4k3** [reply]
But the settings you ask for are the resin settings. You can only define them on Resin settings under lychee.
If not so, provide screenshots on lychee interface where you want that settings

**Comment by webtv123** [comment]
![image](https://github.com/sn4k3/UVtools/assets/61340950/61ca62f9-b1fa-4467-8e15-81f5545ebb20)

**Reply by sn4k3** [reply]
So that's the resin settings page.
You can import your file directly as I shown on first screenshot.

Go to Printers -> Scroll down to Resin - Print Settings -> Select the printer -> Add new resin -> Import from file -> Select the file you want import from

**Comment by webtv123** [comment]
non supported so i converted it in uv tools im not sure if it can be added through resin settings
![image](https://github.com/sn4k3/UVtools/assets/61340950/f6c20399-4530-4d4d-96ed-52063ab90462)

**Reply by sn4k3** [reply]
You don't need to convert, you can simply rename file to pwmx. 
It detect different printer but that's ok, force the Import and check the settings

**Comment by webtv123** [comment]
I'm not sure but I think to burn layer exposure times not correct.
![image](https://github.com/sn4k3/UVtools/assets/61340950/6e45f19f-a58c-4ca6-9b52-4071117857ec)
![image](https://github.com/sn4k3/UVtools/assets/61340950/589e6021-a0d8-483b-b886-4831f1bd23e5)

**Reply by sn4k3** [reply]
You are comparing global settings with layer settings, do not use that to compare. 
Instead go to `Tools - Edit print parameters`. There you have the settings lychee would get.

**Comment by webtv123** [comment]
Okay that looks like the correct settings how do I get this into LycheeSlicer Without having to do it manually one setting at a time

**Reply by sn4k3** [reply]
I think you just did. Didn't you imported the file with the settings?

**Comment by webtv123** [comment]
i see custom does this look correct thank 
![image](https://github.com/sn4k3/UVtools/assets/61340950/9bee8359-9dff-49f0-bc0b-abcc1d2abeb1)
![image](https://github.com/sn4k3/UVtools/assets/61340950/a6e61c18-4ffb-4a37-ad0a-73fab9dd64e5)

**Reply by sn4k3** [reply]
Was that resin profile imported from same file you loaded into UVtools?

**Comment by webtv123** [comment]
see video i just uploaded still in Progress of being uploaded so make sure you go all the way to the end. Trying to import anycubics test print settings into my monoprice printer
https://youtu.be/4FInskOV91s?feature=shared

**Reply by sn4k3** [reply]
You can't rename an anycubic file to ctb! You can rename pm3 to pwmx because they are same file format but never to CTB!
However, something is wrong in the importing, this is something you must talk with lychee, is out of my scope.

**Comment by webtv123** [comment]
Thank you I understand, i noticed all so it will slice as a pm3 but you cant import resin as a .pm3 non supported on resin import
